virtcontainers/clh.go

```diff
@@ -860,12 +859,12 @@ func (clh *cloudHypervisor) getAvailableVersion() error {
 	versionSplit[0] = strings.TrimLeft(versionSplit[0], "v")
 	major, err := strconv.ParseUint(versionSplit[0], 10, 64)
 	if err != nil {
-		return err
+		return fmt.Errorf("Failed to parse cloud-hypervisor version: %s, Unexpected format", words[1])
```

Comment:
- if we get to this error scenario, something unexpected happened so maybe `words[1]` isn't what we expect either. As such, I'd consider logging `words`, not just `words[1]` (same comment for the other errors below).
- To help with debugging, you might want to log `words` when the check on the number of fields above fails too?

Reply:
Agree. I am using words in all error logs now. Thanks.